packages/react-dom/src/client/ReactDOM.js

```diff
@@ -170,15 +170,16 @@ export {
   batchedUpdates as unstable_batchedUpdates,
   flushSync,
   ReactVersion as version,
-  // Disabled behind disableLegacyReactDOMAPIs
   findDOMNode,
+  // Disabled behind disableLegacyReactDOMRenderAPIs
```

Comment:
I think we should remove these from here, and then in the entry points we can conditionally include these from ReactDOMLegacy if the flag is on.